src/app/shared/services/migration/migration.service.ts

```diff
@@ -74,6 +86,16 @@ export class MigrationService {
     }
     return ret;
   }
+
+  private async updateOnboardingServicePreferences() {
+    const preferences = this.preferenceManager.getPreferences(
+      OnboardingService.name
+    );
+    if (!(await preferences.getBoolean('IS_ONBOARDING'))) {
```

Comment:
If `IS_ONBOARDING` (an preference which is only truthy if the user finished onboarding process in old versions) is falsy => set the new onboarding preferences truthy
doesn't seem right